ESCPOS_NET/Emitters/ICommandValues.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace ESCPOS_NET.Emitters
+{
+    public interface ICommandValues
+    {
+        /* Barcodes */
+        byte PrintBarcode { get; }
```

Comment:
One of the downsides of creating an interface like this is we lose the nice grouping of the different command values. For example, before we would reference something like `Barcodes.SetBarWidth`, whereas now it would be `Values.SetBarWidth`. There is no nice indication that this is a value that pertains to Barcodes. 

We could preface the Values with `Cmd` or `Barcodes` so we use it like this: `Values.CmdGS` or `Values.BarcodesSetBarWidth`...but that doesn't feel quite right either. Any suggestions?

Reply:
Perhaps for the sake of simplicity just rename `Values` to `Cmd` or `ByteCmd`, I don't know... Something that can represent all of it.

But again, that's more of a decision that Luke would have to give his opinion about.

Reply:
@lukevp Any thoughts on this?